snap/snapcraft.yaml

```
@@ -0,0 +1,46 @@
+name: foundationdb
+version: 6.0.18
+summary:  An open source, distributed, transactional key-value store.
+description:| 
+A multi-model NoSQL database.
```

```suggestion
The open source, distributed, transactional key-value store.
```

To copy the line from our GitHub repo.

---

That can be changed